changelog/unreleased/visible-links

```diff
@@ -0,0 +1,5 @@
+Enhancement: Display full public and private links
+
+We've added under names of public and private links a full link so that user can copy it without using the copy to clipboard button.
```

Comment:
```suggestion
Below the names of public and private links we've added the respective full URL so that users can copy it without the copy to clipboard button.
```